README.md

```diff
@@ -66,6 +66,24 @@ Flags:
       --name string   Name of e2e cluster. (default "kind")
 ```
 
+### Get cluster's kubeconfig file path
+
+- Outputs the kubeconfig file path.
+
+```bash
+$ e2ectl kubeconfig file-path -h
```

Comment:
One late comment from me: I'd prefer to replace `file-path` to `path` to make it less cumbersome to type.